CHANGES.md

```diff
@@ -1,5 +1,8 @@
 ## Changelog
 
+### 1.79.1 (2019-05-03)
```

Comment:
Why not 1.80.0?

Reply:
Discussed F2F, if the policy is minor bug fix == patch, then I presume new feature = minor, and major functional (breaking) change is major.  Can you add to the README?

Reply:
Added to releasing.md_